mobify/progressive-web-scaffold
# Fix login submission

This PR fixes error handling and submission for the login and create account pages

 **JIRA**: n/a
 **Linked PRs**: n/a

## Changes
- Updated login & registration redux forms to correctly use handleSubmit
- Updated submission promise rejections to send errors to redux form

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Open the menu
- Click "Sign In"
- Confirm the sign in form works as expected (successful login and error states)
- Log out and navigate to the create account page
- Confirm the create account form works as expected (successful account creation and error states)
